src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala

```diff
@@ -147,6 +147,29 @@ class MerklePatriciaTreeIntegrationSuite extends FunSuite
       else assert(rootHash.take(4) == "da8a" && rootHash.drop(rootHash.length - 4) == "0ca4")
     }
   }
+
+  ignore("MPT benchmark") {
```

Comment:
I'm not about sure the `ignore`. I would prefer creating a new sbt task `benchmark` (or `longrunning`) and don't run them on CI. There are other tests that should be included in this category too.

@mcsherrylabs what do you think?

Reply:
If this is the third (or more than third) 'benchmark' task, then yes, we should move them all out to another grouping. I further assume moving this won't lower the unit test coverage.